.circleci/config.yml

```diff
@@ -402,6 +402,17 @@ jobs:
             RUST_BACKTRACE=1 $CARGO $CARGOFLAGS test \
               --features='vanilla' \
               --no-default-features
+  compile-benches:
```

Comment:
@huitseeker this defines the job, but it will not run in CI.  To make it run, it needs to be part of the commit-verify workflow https://github.com/libra/libra/blob/96774332c59edf678a8813cca83996aba584417c/.circleci/config.yml#L634-L635

Reply:
Thanks!